apps/theyworkforyou/components/AnswerChoice.vue

```diff
@@ -68,7 +68,13 @@ export default {
   computed: {
     getColor() {
       if (this.hover) {
-        return '#00EAA4' // green
+        if(this.choice_id === 'approve') {
```

Comment:
ส่วนตัวถ้าต้อง if else + return ตามค่าที่เป็นไปได้ของตัวแปรเดียวกัน (ในเคสนี้คือ `this.choice_id`) ผมชอบใช้ switch case เพราะอ่านง่ายดี

```js
switch(this.choice_id) {
  case 'approve':
     return '#00EAA4';
  case 'disprove':
    ....
}
```
อันนี้แลกเปลี่ยนไอเดียเฉยๆ นะ เลือกตามที่ชอบได้เลย